fishtank/src/cluster.ts

```diff
@@ -51,20 +57,53 @@ export class Cluster {
       labels: { [CLUSTER_LABEL]: this.name },
     }
 
+    const args: string[] = []
```

Comment:
You can set this to `['start']`, so that you can remove the logic below:

```typescript
    if (args.length > 0) {
      runOptions.args = ['start'].concat(args)
    }
```

Reply:
I still need the if logic like this
```
if (args.length > 0) {
      runOptions.args = args
    }
```